doc/source/whatsnew/v0.18.1.txt

```diff
@@ -72,15 +72,81 @@ API changes
 
 
 - ``CParserError`` is now a ``ValueError`` instead of just an ``Exception`` (:issue:`12551`)
+
 - ``pd.show_versions()`` now includes ``pandas_datareader`` version (:issue:`12740`)
 
+- Using ``apply`` on resampling groupby operations (e.g. ``df.groupby(pd.TimeGrouper(freq='M', key='date')).apply(...)``) now has the same output types as similar ``apply``s on other groupby operations (e.g. ``df.groupby(pd.Grouper(key='color')).apply(...)``). (:issue:`11742`).
+
+Previous behavior:
+
+.. code-block:: python
+
+    In [1]: df = pd.DataFrame({'date': pd.to_datetime(['10/10/2000', '11/10/2000']), 'value': [10, 13]})
+
+    In [2]: df.groupby(pd.TimeGrouper(key='date', freq='M')).apply(lambda x: x.value.sum())
+    Out[2]:
+    ...
+    TypeError: cannot concatenate a non-NDFrame object
+
+    In [3]: df.groupby(pd.TimeGrouper(key='date', freq='M')).apply(lambda x: x[['value']].sum())
+    Out[3]:
+    date
+    2000-10-31  value    10
+    2000-11-30  value    13
+    dtype: int64
+
+    In [3]: type(df.groupby(pd.TimeGrouper(key='date', freq='M')).apply(lambda x: x[['value']].sum()))
```

Comment:
I don't think you *also* need to show the types, instead put a comment right above it

```
# output is a Series
df.groupby(.......)


# output is a DataFrame
df.groupby(....)
```

this will shorter up this section a bit. (just 3 examples then in the previous block), you can put the same comments in the new block.

Imagine you are the user reading this and try to be as simple (but as complete) as possible.